--- CorsixTH/CMakeLists.txt	2015-01-15 19:36:05.369926373 +0800
+++ CorsixTH/CMakeLists.txt	2015-01-15 19:37:30.019925740 +0800
@@ -200,10 +200,10 @@
     FIXUP_BUNDLE(${CMAKE_INSTALL_PREFIX}/CorsixTH.app \"\" \"\")
     ")
 ELSE()
-  install(TARGETS CorsixTH RUNTIME DESTINATION CorsixTH)
+  install(TARGETS CorsixTH RUNTIME DESTINATION ${BINDIR})
   install(DIRECTORY Lua Levels DESTINATION CorsixTH PATTERN "*.svn" EXCLUDE)
   install(DIRECTORY Bitmap DESTINATION CorsixTH
         FILES_MATCHING REGEX ".*\\.(tab|pal|dat|png)$"
         PATTERN "*.svn" EXCLUDE)
-  install(FILES CorsixTH.lua ../LICENSE.txt CorsixTH.ico DESTINATION CorsixTH )
+  install(FILES CorsixTH.lua CorsixTH.ico DESTINATION CorsixTH )
 ENDIF()

--- MapEdit/CMakeLists.txt	2015-01-15 19:39:21.986591570 +0800
+++ MapEdit/CMakeLists.txt	2015-01-15 19:39:42.273258086 +0800
@@ -197,5 +197,5 @@
     FIXUP_BUNDLE(${CMAKE_INSTALL_PREFIX}/MapEdit.app \"\" \"\")
     ")
 ELSE()
-  install(TARGETS MapEdit RUNTIME DESTINATION CorsixTH)
+  install(TARGETS MapEdit RUNTIME DESTINATION ${BINDIR})
 ENDIF()

--- CMakeLists.txt	2015-01-15 19:41:42.066590522 +0800
+++ CMakeLists.txt	2015-01-15 19:42:37.966590105 +0800
@@ -30,6 +30,8 @@
   SET(CMAKE_OSX_ARCHITECTURES x86_64)
 ENDIF()
 
+set(BINDIR bin CACHE PATH "where to install binaries")
+
 # Define our options
 OPTION(WITH_SDL "Activate SDL Renderer" ON) # our default option
 OPTION(WITH_AUDIO "Activate Sound" ON) # enabled by default
