--- configure.ac	2013-08-14 07:31:54.967212655 +0800
+++ configure.ac.new	2013-08-14 07:34:11.003142351 +0800
@@ -101,7 +101,10 @@
 # ==================
 # Check dependencies
 # ==================
-PKG_CHECK_MODULES([SDL],[sdl >= 1.2.13 SDL_mixer >= 1.2.11 SDL_gfx >= 2.0.22 SDL_image >= 1.2])
+PKG_CHECK_MODULES([SDL],[sdl >= 1.2.13 SDL_mixer >= 1.2.11 SDL_gfx >= 2.0.22 SDL_image >= 1.2],,
+[
+	PKG_CHECK_MODULES([SDL],[sdl >= 1.2.13 SDL_mixer >= 1.2.11 SDL_gfx >= 2.0.22 SDL2_image >= 2.0])
+])
 PKG_CHECK_MODULES([YAML],[yaml-cpp <= 0.3.0])
 AX_CHECK_GL
 
